internal/pkg/agent/cmd/container.go

```diff
@@ -126,14 +126,15 @@ be used when the same credentials will be used across all the possible actions a
   KIBANA_FLEET_USERNAME - kibana username to enable Fleet [$ELASTICSEARCH_USERNAME]
   KIBANA_FLEET_PASSWORD - kibana password to enable Fleet [$ELASTICSEARCH_PASSWORD]
   KIBANA_CA - path to certificate authority to use with communicate with Kibana [$ELASTICSEARCH_CA]
+  ELASTIC_AGENT_TAGS - user provided tags for the agent
```

Comment:
Can you add the expected format, is this a comma-separated list?

Reply:
Should it be prefixed with `ELASTIC_AGENT`? Are the tags useful in standalone mode? If not I think that `FLEET_TAGS` would be better. As its the tags that will be show for this Elastic Agent in Fleet.

Reply:
We are starting to have so many tags with a different prefixes, @nimarezainia I would like your opinion on this.

Reply:
I thought tags apply for enrolled agents only (can I get a confirmation here @nimarezainia?)
I chose the `ELASTIC_AGENT_ ` naming as the tag applies to the specific agent enrolled and not to the Fleet Server, agent is the subject receiving the tags, that said, I am open to any naming.

Reply:
these tags only apply to enrolled agents (managed agents) "ELASTIC_AGENT_TAGS" sounds right to me per @narph explanation.